packages/cf.js/src/index.ts

```diff
@@ -1,4 +1,6 @@
-import NodeProvider from "@counterfactual/node-provider";
+import NodeProvider, {
```

Comment:
Seems like a weird pattern to import these from another package then export them?

Reply:
That's because we want CF.js users _not_ to install NodeProvider. CF.js bundles this package so any dapp developers only need to install CF.js instead of both packages. It's for better developer ergonomics, IMO.

Reply:
In that case, what's the disadvantage of folding the node-provider code to be a subset of cf.js package?

Reply:
That would prevent potential reuse of this package from `cf-wallet.js`.

Reply:
Hmm fair enough. I'll put up a PR to introduce a README for the `node-provider` package in that case to clarify this pattern.